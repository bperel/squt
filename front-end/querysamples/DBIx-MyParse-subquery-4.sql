SELECT field2 FROM table2 WHERE field3 IN (SELECT field1 FROM table1 WHERE cond1)
