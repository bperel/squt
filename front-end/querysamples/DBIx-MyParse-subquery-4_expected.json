{
   "Subqueries" : {
      "0" : {
         "SubqueryAlias" : "0",
         "SubqueryType" : "IN_SUBS",
         "Tables" : {
            "table1" : {
               "table1" : {
                  "OUTPUT" : {
                     "field1" : {
                        "-1" : "field1"
                     }
                  },
                  "CONDITION" : {
                     "cond1" : {
                        "EXISTS" : "1"
                     }
                  }
               }
            }
         }
      }
   },
   "Tables" : {
      "table2" : {
         "table2" : {
            "OUTPUT" : {
               "field2" : {
                  "-1" : "field2"
               }
            },
            "CONDITION" : {
               "field3" : {
                  "IN_SUBS" : "0"
               }
            }
         }
      }
   },
   "Warning" : {
      "No alias" : {
         "subquery #0" : ""
      }
   }
}
